app/components/collections/edit/work_type_component.html.erb

```diff
@@ -1,35 +1,37 @@
 <div data-controller="work-type">
   <div class="pane-section">
     <%= render Elements::Forms::FieldsetComponent.new(label: t('collections.edit.fields.work_type.label'), tooltip: t('collections.edit.fields.work_type.tooltip_html')) do %>
+    <%= tag.p class: 'form-text mb-3' do %><%= type_help_text %><% end %>
```

Comment:
Are you sure this needs the `form-text` class?

Also, per our (very loose) conventions, this should be a `<p>` instead of `tag.p` since there are no variables / other ruby.)